crates/release_plz_core/src/release.rs

```diff
@@ -30,10 +31,12 @@ pub struct ReleaseRequest {
     pub token: Option<SecretString>,
     /// Perform all checks without uploading.
     pub dry_run: bool,
-    /// Publishes GitHub release.
+    /// Publishes GitHub/Gitea release.
     pub git_release: Option<GitRelease>,
-    /// GitHub repo URL.
+    /// GitHub/Gitea repo URL.
     pub repo_url: Option<String>,
+    /// Kind of git host where your project is hosted
+    pub backend: GitBackend,
```

Comment:
I think it's easier to move this field into the `GitRelease` field. They are connected.